benchmark_runner/workloads/workloads_operations.py

```diff
@@ -18,6 +18,7 @@
 
 
 class WorkloadsOperations:
+    oc = None
```

Comment:
@kpouget, What the purpose of 'oc = None' parameter if you are not using it. can you explain ?